docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -30,66 +30,79 @@ Each service of the router pipeline (`router`, `supergraph`, `subgraph`) has its
+| `subgraph_response_errors`  | Yes         |                  | Json Path into the subgraph response body errors (it might impact performance) |
+| `subgraph_request_header`   | Yes         |                  | The name of a subgraph request header                                          |
+| `subgraph_response_header`  | Yes         |                  | The name of a subgraph response header                                         |
+| `subgraph_response_status`  | Yes         | `code`\|`reason` | The status of a subgraph response                                              |
+| `supergraph_operation_name` | Yes         | `string`\|`hash` | The operation name from the supergraph query                                   |
+| `supergraph_operation_kind` | Yes         | `string`         | The operation kind from the supergraph query                                   |
+| `supergraph_query`          | Yes         | `string`         | The graphql query to the supergraph                                            |
+| `supergraph_query_variable` | Yes         |                  | The name of a supergraph query variable                                        |
+| `request_context`           | Yes         |                  | The name of a request context key                                              |
+| `response_context`          | Yes         |                  | The name of a response context key                                             |
+| `baggage`                   | Yes         |                  | The name of a baggage item                                                     |
+| `env`                       | Yes         |                  | The name of an environment variable                                            |
+| `static`                    | No          |                  | A static string value                                                          |
+| `error`                     | No          | `reason`         | a string value containing error reason when it's a critical error              |
+
+#### GraphQL
+
+GraphQL metrics are extracted from the response data being returned to the client.
+
+| Selector         | Defaultable | Values           | Description                                 |
+|------------------|-------------|------------------|---------------------------------------------|
+| `list_length`    | No          | `value`          | The length of a list from the response data |
+| `field_name`     | No          | `string`         | The name of a field from the response data  |
+| `field_type`     | No          | `string`         | The type of a field from the response data  |
+| `type_name`      | No          |                  | The graphql from the response data          |
+| `operation_name` | Yes         | `string`\|`hash` | The operation name of the query             |
+| `static`         | No          |                  | A static string value                       |
```

Comment:
Perhaps just my ignorance about metrics/telemetry: do these apply to _any_ list/field/type etc. in the response data?

Reply:
Yes, these are metrics around response data where we tie the data back to the graphql schema.